src/components/victory-group/victory-group.js

```diff
@@ -404,6 +411,7 @@ export default class VictoryGroup extends React.Component {
         theme: child.props.theme || props.theme,
         labelComponent: props.labelComponent || child.props.labelComponent,
         style,
+        domainPadding: {x: (props.width / noOfChildren) / 2},
```

Comment:
I think this might need to be something more like `xOffset * childComponents.length / 2`